ci/gpuci/run.sh

```diff
@@ -22,4 +22,7 @@ fi
 conda config --set ssl_verify false
 conda install -k conda-build conda-verify anaconda-client
 conda build conda/recipe
-anaconda -t ${MY_UPLOAD_KEY} upload -u ${CONDA_USERNAME:-gpuci} --label main --skip-existing /opt/conda/conda-bld/${ARCH_DIR}/gpuci-tools*.tar.bz2
+anaconda -t ${MY_UPLOAD_KEY} upload -u ${CONDA_USERNAME:-gpuci} --label main --skip-existing \
+  /opt/conda/conda-bld/${ARCH_DIR}/gpuci-tools*.conda \
+  /opt/conda/conda-bld/${ARCH_DIR}/gpuci-tools*.tar.bz2 \
+;
```

Comment:
```suggestion
anaconda -t ${MY_UPLOAD_KEY} upload -u ${CONDA_USERNAME:-gpuci} --label main --skip-existing /opt/conda/conda-bld/${ARCH_DIR}/gpuci-tools*.{conda, tar.bz2}
```

Think it'd be a bit simpler to just use `{option1,option2}` instead of repeating the search directory twice.

I'd still support breaking that onto another line if you want, but don't feel as strongly about it.

Reply:
Honestly was thinking of replacing this by `find` as currently `shellcheck` doesn't like the lack of quotes

Reply:
that's fine too, whatever works 🤷🏻

Reply:
Yeah it was just CI was failing due to `shellcheck`. So that also needed to be addressed

Though do appreciate the tip

Please let me know what you think of the rewrite